release/Makefile

```diff
@@ -124,6 +123,7 @@ host: $(OUTDIR)/opam-full-$(VERSION).tar.gz build/$(HOST).env
 	  echo "$(call LINKING,$(HOST_OS))" >src/client/linking.sexp && \
 	  $(MAKE) opam; \
 	)
+	chmod u+w build/opam-full-$(VERSION)/_build/default/src/client/opamMain.exe
```

Comment:
Similarly here: in fact, the duplication kinda increases the case for having `$(MAKE) opam-stripped`?

Reply:
done in https://github.com/ocaml/opam/pull/6208